modules/apps/depot/depot-web/src/main/java/com/liferay/depot/web/internal/item/selector/provider/AssetLibraryGroupItemSelectorProvider.java

```diff
@@ -0,0 +1,40 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.depot.web.internal.item.selector.provider;
+
+import com.liferay.depot.constants.DepotConstants;
+import com.liferay.item.selector.provider.GroupItemSelectorProvider;
+
+import org.osgi.service.component.annotations.Component;
+
+/**
+ * @author Roberto Díaz
+ */
+@Component(service = GroupItemSelectorProvider.class)
+public class AssetLibraryGroupItemSelectorProvider
+	extends BaseGroupItemSelectorProvider {
+
+	@Override
+	public String getGroupType() {
+		return "asset-library";
+	}
```

Comment:
Previously, `getGroupType` returned a string that had a 1:1 mapping to `group.getType()`, but that's no longer true. I think Brian will not like this as is; maybe, to avoid changing API, we can try using more specific type constants, like "asset-library-depot" and "space-depot", so that at least it includes the group type (depot) in the enum.